ddr/tools/getmacros

```diff
@@ -215,8 +221,17 @@ parse_namespace_policy_files() {
 			mv ${f} ${f}.orig
 			cp ${f}.orig ${f}
 
+			# Increment the sequence number embedded in generated include guard names.
+			(( filenum++ ))
+
 			# read from the backup, rather than the file to which we're appending
-			process_one_policy_file < ${f}.orig >> ${f}
+			if [ $zos390 = yes ] ; then
```

Comment:
Is this necessary? Since all the output is done via `echo` it should already be in EBCDIC (unless we have some kind of ASCII version of echo on the build path

Reply:
It would seem reasonable to think that, but `bash` seems to produce ASCII. Without using `iconv` the header files read to produce `*.i` would be a mix of ASCII and EBCDIC.

Reply:
Indeed it does seem to produce ASCII output:

```
JENKINS:/tmp: >echo "echo \"Testing encoding when piping to file\"" > test-encoding.sh
JENKINS:/tmp: >chmod +x test-encoding.sh
JENKINS:/tmp: >chtag -p test-encoding.sh
t ISO8859-1   T=on  test-encoding.sh
JENKINS:/tmp: >bash test-encoding.sh
Testing encoding when piping to file
```

I don't know enough about DDR but not sure why this would matter though? Because as long as `_BPXK_AUTOCVT=ON` is set, which it should be on z/OS then reading from disk will always be auto-converted to EBCDIC.

Reply:
The issue is because it is tacking the output on to the end of an existing file. (although it would be nice if the os was smart enough to auto convert when appending to a tagged file)

Reply:
> (although it would be nice if the os was smart enough to auto convert when appending to a tagged file)

It is. Appending to a tagged file appends in the same format:

```
JENKINS:/tmp: >bash ./test-encoding.sh >> out
JENKINS:/tmp: >bash ./test-encoding.sh >> out
JENKINS:/tmp: >cat out
Testing encoding when piping to file
Testing encoding when piping to file
JENKINS:/tmp: >chtag -p out
t ISO8859-1   T=on  out
```

I'm still not sure what the issue seems to be. Is the existing original file not properly tagged to begin with?

Reply:
`getmacros` is not as simple as these examples; the body of `process_one_policy_file` is executed in a sub-shell due to the use of `sed ... | while read ...`.

Reply:
Are you able to provide a simplistic example of how we end up in a situation where conversion is needed? If the on disk files are properly tagged from the start I don't think conversion should be needed to read or write. Everything should just work™.

Reply:
There's quite a mix of files; here's a sampling of files _not_ tagged as EBCDIC:
```
m IBM-1047    T=off vm/bcutil/ut_j9bcu.h
- untagged    T=off vm/dfix/dfix.h
t ISO8859-1   T=on  vm/include/openj9_version_info.h
- untagged    T=off vm/nls/j9vmnls.h
m IBM-1047    T=off vm/oti/vmhook.h
- untagged    T=off vm/oti/j9vmconstantpool.h
```
A number of tools need to be fixed to completely clean this up.

Reply:
Ok, so this is the source of the problem. I suspect those files are generated as I can't find them in the repository. Whatever is generating them needs to be fixed to generate properly file-tagged files. After that is done we no longer need the ASCII/EBCDIC conversions here.

I agree, this is outside the scope of this PR. Thanks for the explanations!